src/intl/en/page-contributing-translation-program-contributors.json

```diff
@@ -5,5 +5,6 @@
   "page-contributing-translation-program-contributors-our-translators-2": "With thousands of community members contributing translations to our project, it is difficult to acknowledge everyone.",
   "page-contributing-translation-program-contributors-our-translators-3": "All translators are listed alphabetically based on their chosen name in Crowdin. If you are a translator and would like to use your real name, nickname, ENS domain, etc., you can change your Full Name in Crowdin.",
   "page-contributing-translation-program-contributors-meta-title": "Our Translations",
-  "page-contributing-translation-program-contributors-meta-description": "A list of our translation contributors."
+  "page-contributing-translation-program-contributors-meta-description": "A list of our translation contributors.",
+  "page-contributing-translation-program-contributors-total-contributors": "people have contributed to ethereum.org to date!"
```

Comment:
I think doing the translation this way might cause grammar problems in other languages? @lukassim